File: web-dev/cs50/intro-comp-science/Problem-sets/pset6/mario_less/mario.py
from cs50 import get_int


## Starting with our main program
def main():
    while True:
        ## Getting an integer input from the user defined as the height
        height = get_int("Height:")

        ## Setting a condition that the user's input must fulfill
        if height > 0 and height <= 8:
            break

        ## Calling our function that will draw the pyramid
    mario(height)

## Defining a function that draws the pyramid
def mario(height):
    for i in range(height):
        print(' '*(height-i-1)+'#'*(i+1))

main()

File: web-dev/cs50-2021/C-Program/week1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>


// This is the mario-less program 


int main(void)
{
 // Prompt user for an input
 int height;
 do
 {
  /* code */
  height = get_int("Height of for the Pyramid: ");
 } while (height < 1 || height > 8);

 // Iteration for columns up until the given input by the user
 for (int i = 0; i < height; i++)
 {
  // This will print an empty spaces helping to realign our blocks to the right
  for (int j = (height - i) - 1; j >= 1; j--)
  {
   printf(" ");
  }

  // Printing the HASHES
  for (int k = 0; k <= i; k++)
  {
   printf("#");
  }

  // Move to the next line after each execution
  printf("\n");
 }
 
}